src/Client/StorePaymentMethodLightningNetwork.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
+use BTCPayServer\Http\CurlClient;
+
+/**
+ * Handles a stores LightningNetwork payment methods.
+ *
+ * @see https://docs.btcpayserver.org/API/Greenfield/v1/#tag/Store-Payment-Methods-(Lightning-Network)
+ */
+class StorePaymentMethodLightningNetwork extends AbstractStorePaymentMethodClient
+{
+    public function getPaymentMethods(string $storeId): array
```

Comment:
Add PHPdoc:
@return \BTCPayServer\Result\StorePaymentMethodLightningNetwork[]
this way IDEs like PHPStorm will know the return type is an array of this type of objects.

Reply:
done